tools/cheminformatics/modal.py

```diff
@@ -1,17 +1,16 @@
 from modal import App, web_endpoint
 from modal import Image
 
-rdkit_image = (
-    Image.debian_slim(python_version="3.12")
-    .pip_install("rdkit")
-)
+rdkit_image = Image.debian_slim(python_version="3.12").pip_install(["rdkit", "numpy"])
```

Comment:
Would it be good to fix the versions of the packages? Numpy just had a major update (and at least in my course there were a lot of projects with breaking changes).